tests/ShortidTest.php

```diff
@@ -79,6 +79,6 @@ public function testJsonEncode()
 	{
 		$generated = Shortid::generate();
 
-		$this->assertTrue((string) $generated === json_encode($generated));
+		$this->assertSame('"'.(string) $generated.'"', json_encode($generated));
```

Comment:
What's the purpose of concatenating quotes?

Reply:
json_encode('dfdsfsd') === '"dfdsfsd"';

Reply:
So, what's the purpose of casting to string?

Reply:
And as to me to receive a string instead of a class, it is possible to cause certainly method __toString without casting.

Reply:
Well, you could also assert that `$generated` is the same as `$generated`.
But it doesn't make sense

Reply:
When I took your library, I did not expect that the class would be the generated value.
And when this class was assigned to the property of the object, and after that the object was serialized through the json_encode at the output, it turned out:

```json
{"test":{}}
```

Therefore, I suggested adding an implementation of the interface JsonSerializable and thus got the result::

```json
{"test":"chZ7LjD"}
```

Reply:
I understand the purpose of JsonSerializable. Anyway, casting to string a variable that is concatenated to another string is still not needed, since PHP is automatically converting such variable

Reply:
I give up, I do not understand how the test should look like.

Reply:
`$this->assertSame('"'.$generated.'"', json_encode($generated));`